python/cudf/cudf/core/udf/nrt_utils.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+
```

Comment:
@gmarkall would you be willing to give this file and the usage of it's contents a look? The intent here is a toggle for NRT while we figure out more general refcount pruning upstream.

Reply:
cc @vyasr also, the assumption here is that NRT can be turned on and off dynamically via the config for a single code generation run, see https://github.com/NVIDIA/numba-cuda/blob/66a7b6fb807418715b9977b26095822a9bad5d8f/numba_cuda/numba/cuda/target.py#L109

Reply:
Understood, so numba-cuda will check the env var every time we hit that point.